assets/app/views/_overview-deployment.html

```diff
@@ -78,7 +78,7 @@
     </div>
 
     <!-- Pods donut chart and scaling controls -->
-    <div column>
+    <div column class="overview-pods">
```

Comment:
I'm inclined to think we should wrap up the `scaling-controls` in a directive as well, rather than have the details for the control live in the `overviewDeployment` template & controller.

Its probably good to keep the `pod-status-chart` directive as it is, but perhaps create a `pod-scaling-control` directive to wrap up this markup:

```html
<!-- pod-scaling-control tpl -->
<div row>
  <div column>
   <!-- pass in this info as attrs on the directive -->
    <pod-status-chart
        chart-id="rc-{{rc.metadata.name}}-donut-chart"
        pods="pods"
        desired="desiredReplicas || rc.spec.replicas || 1"
        ng-click="viewPodsForDeployment(rc)">
    </pod-status-chart>
  </div>

  <div column class="scaling-controls">
    <!-- spacer -->
    <div flex></div>
    <div column>
      <a href="" ng-click="scaleUp()" title="Scale up">
        <i class="fa fa-chevron-up"></i>
        <span class="sr-only">Scale up</span>
      </a>
      <a href="" ng-click="scaleDown()" title="Scale down">
        <i class="fa fa-chevron-down"></i>
        <span class="sr-only">Scale down</span>
      </a>
    </div>
    <!-- spacer -->
    <div flex></div>
  </div>
</div>
```

The `pod-scaling-control` directive could look something like this, so it can talk to its controller:

```html
<pod-scaling-control
   up="scaleUp()"
   down="scaleDown()"
   chart-click="viewPodsForDeployment(rc)"
   desired="desiredReplicas || rc.spec.replicas || 1"
   chart-id="rc-{{rc.metadata.name}}-donut-chart">
</pod-scaling-control>
```
and take the fns as attribs on the directive && also hide the `_.debounce` detail in the directive (perhaps even make that configurable on the directive `delay="500"` or separately `up-delay="500" down-delay="1000"` ).